doc/howtos/backend.rst

```diff
@@ -1708,13 +1708,22 @@ server with the library ``xmlrpc.client``::
 JSON-RPC Library
 ----------------
 
+.. danger::
```

Comment:
danger is for the very important things like security related, here this does not even deserve a warning, just a note in the "The following example..." is probably enough.
Also, try to keep 80 chars max for the doc lines.